x-pack/plugins/rule_registry/common/assets/field_maps/ecs_field_map.ts

```diff
@@ -3741,6 +11193,11 @@ export const ecsFieldMap = {
     array: false,
     required: false,
   },
+  'threat.enrichments.indicator.file.hash.sha384': {
```

Comment:
@ecezalp or @kgeller . Kevin's updating the rule registry with the most recent ecs changes. Just want to make sure these other fields like `threat.*` should be in here? Thanks!

Reply:
@michaelolo24 I think it's fine. There have been other schema changes in between when this file was last updated and the linux event model changes, and so that's what you are seeing.

I am however a little concerned about https://github.com/elastic/ecs/pull/1842#pullrequestreview-916349079 ...

@ebeahan is working getting that filtering PR out there now, which will remove a decent amount of these new fields. This is related to something I discussed with @mitodrummer a couple of days ago. In ECS, the re-uses chain, and that means that some of the new re-uses your team has added, are getting fields that you don't necessarily want or need. And therefore that subset filtering will help us to clean all that up.

Now I don't know that you need to be held up by us yet again, but I want to bring that `*` here, that we anticipate a decent chunk of these fields will not actually end up in ECS.